pom.xml

```
@@ -15,7 +15,7 @@
         <resteasy.version>6.0.3.Final</resteasy.version>
         <guice.version>5.1.0</guice.version>
         <oauth-client.version>1.33.3</oauth-client.version>
-        <jackson.version>2.13.3</jackson.version>
+        <jackson.version>2.13.4</jackson.version>
         <jackson-databind.version>2.13.3</jackson-databind.version>
```

```suggestion
        <jackson-databind.version>2.13.4</jackson-databind.version>
```
The `jackson` and `jackson-databind` versions should be updated at the same time, @Morpheu5, unless databind is ahead by a patch (which it sometimes must be, hence the separate variable).

---

> The `jackson` and `jackson-databind` versions should be updated at the same time, @Morpheu5, unless databind is ahead by a patch (which it sometimes must be, hence the separate variable).

Well spotted, I don't see any reasons why this shouldn't be the case.